src/main/java/frc/robot/Robot.java

```diff
@@ -177,6 +177,10 @@ public void teleopInit() {
     @Override
     public void robotPeriodic() {
         CommandScheduler.getInstance().run();
+        SmartDashboard.putNumber("TargetX", Pi.getTargetCenterX());
+        SmartDashboard.putNumber("TargetY", Pi.getTargetCenterY());
+        SmartDashboard.putNumber("CargoX", Pi.getCargoCenterX());
+        SmartDashboard.putNumber("CargoY", Pi.getCargoCenterY());
```

Comment:
Wanted to have the actual value we use on the dashboard to see what the robot is aiming at.